index.bs

```diff
@@ -5741,7 +5741,7 @@ which just allows the additional value <emu-val>null</emu-val>
 to be a member of its set of values.  [=Nullable types=]
 are represented in IDL by placing a <span class="char">U+003F QUESTION MARK ("?")</span>
 character after an existing type.  The inner type must not
-be {{any}},
+be {{any}}, a [=Promise type=],
 another nullable type, or a [=union type=]
```

Comment:
This looks incorrect. If Promise is non-nullable, the next cannot be "another nullable".